salt/bugs/init.sls

```diff
@@ -4,6 +4,19 @@ include:
   - tls.lego
   - nginx
 
+/etc/consul.d/service-bugs.json:
```

Comment:
should this be moved inside the `for tracker` loop to register a service for each of the trackers, mapping to the name used in the haproxy config?

Reply:
[98a43e4](https://github.com/python/psf-salt/pull/393/commits/98a43e4435031274223870a854656f9229055ea3)